samtranslator/model/stepfunctions/events.py

```diff
@@ -28,6 +28,8 @@ class EventSource(ResourceMacro):
     # TODO: Make `EventSource` an abstract class and not giving `principal` initial value.
     principal: str = None  # type: ignore
 
+    Target: Optional[Dict[str, str]]
```

Comment:
What's the benefit of using a dict over a str? It seems that we only want a `target_id`.

Reply:
It's to match what we have here: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-function-target.html